client/bugLogClientAngular.js

```diff
@@ -1,134 +1,171 @@
+/**
+ * This module requires stacktrace-js and angular, it has been tested with:
+ *
+ * "stacktrace-js": "~0.6.4"
+ * "angular": "~1.3.15"
+ */
+
 angular.module('BugLogHq', [])
-//the application to tell BugLog from where the report is coming from.
-.constant('appName', "app")
-.constant('bugLogConfig', {
-    // this is the location of the BugLog server that will be receiving the error reports (needs to be a REST/POST endpoint)
-    listener: "",
-    // the hostname to tell BugLog from where the report is coming from. Leave empty to get the info from the browser.
-    hostName: "",
-    // If the BugLog server requires an API Key to talk to it, then here is where you put it.
-    apiKey: ""
-})
-.config(['$provide',
-    function ($provide) {
-        $provide.decorator('$exceptionHandler', ['$delegate', '$log', 'BugLogService',
-            function ($delegate, $log, BugLogService) {
-                return function(exception, cause) {
-                    //Custom error handling.
-                    try {
-                        BugLogService.notifyService({
-                            message: cause,
-                            error: exception
-                        });
-                    }
-                    catch (e) {
-                        $log.warn("BugLog Service: Error logging bug");
-                        $log.warn(e);
-                    }
-                    // Calls the original $exceptionHandler.
-                    $delegate(exception, cause);
-                };
-            }
-        ]);
-    }
-])
-.factory('BugLogService', ["$window", "$document", "appName", "bugLogConfig",
-    function BugLogService($window, $document, appName, bugLogConfig) {
-        var BugLog = {
+    //the application to tell BugLog from where the report is coming from.
+    .constant('appName', "app")
+    .constant('bugLogConfig', {
+        // this is the location of the BugLog server that will be receiving the error reports (needs to be a REST/POST endpoint)
+        listener: "",
```

Comment:
I noticed that the option to override the HostName is removed. I know there are multiple cases in which users prefer to set the hostname explicitly. If not present, BugLogHQ will revert to use the CGI information, which may or may not be what you want.